### ISSUE TITLE
App crashes on reinstalling app

### ISSUE THREAD

**Issue by anhanh11001** [fossasia/badge-magic-android#253]
**Describe the bug**
I'm updating the latest version of the app by deleting and reinstalling it, and the app constantly crashes with this log:
```
    Process: com.nilhcem.blenamebadge, PID: 23767
    java.lang.ExceptionInInitializerError
        at com.nilhcem.blenamebadge.ui.main.MainActivity.inject(MainActivity.kt:85)
        at com.nilhcem.blenamebadge.ui.main.MainActivity.onCreate(MainActivity.kt:56)
        at android.app.Activity.performCreate(Activity.java:7383)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1218)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3256)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3411)
        at android.app.ActivityThread.-wrap12(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1994)
        at android.os.Handler.dispatchMessage(Handler.java:108)
        at android.os.Looper.loop(Looper.java:166)
        at android.app.ActivityThread.main(ActivityThread.java:7529)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:245)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:921)
     Caused by: java.lang.IllegalStateException: files must not be null
        at com.nilhcem.blenamebadge.util.StorageUtils.getAllFiles(StorageUtils.kt:42)
        at com.nilhcem.blenamebadge.data.fragments.FilesDAO.<init>(FilesDAO.kt:11)
        at com.nilhcem.blenamebadge.data.fragments.FilesContainer.<init>(FilesContainer.kt:5)
        at com.nilhcem.blenamebadge.data.fragments.FilesContainer.<init>(FilesContainer.kt:3)
        at com.nilhcem.blenamebadge.data.fragments.FilesContainer$Companion.getInstance(FilesContainer.kt:15)
```

**Would you like to work on the issue?**
Yes

**Comment by adityastic** [comment]
Because Saved Fragment is not done yet. Will be fixed by today when #236 will be completed

**Comment by adityastic** [comment]
Hey @anhanh11001 ,I assume I can fix this issue. Do you want me to?
